ci/ray_ci/container.py

```diff
@@ -0,0 +1,84 @@
+import os
+import subprocess
+
+from typing import List
+
+from ci.ray_ci.utils import chunk_into_n
+
+DOCKER_ECR = "029272617770.dkr.ecr.us-west-2.amazonaws.com"
+DOCKER_REPO = "ci_base_images"
+DOCKER_TAG = f"oss-ci-build_{os.environ.get('BUILDKITE_COMMIT')}"
+
+
+def run_tests(
+    test_targets: List[str],
+    pre_run_commands: List[str],
+    parallelism,
+) -> int:
+    """
+    Run tests parallelly in docker
+    """
+    chunks = chunk_into_n(test_targets, parallelism)
+    runs = [_run_tests_in_docker(chunk, pre_run_commands) for chunk in chunks]
+    exits = [run.wait() for run in runs]
+    return 0 if all(exit == 0 for exit in exits) else 1
+
+
+def _run_tests_in_docker(
+    test_targets: List[str],
+    pre_test_commands: List[str],
+) -> subprocess.Popen:
+    bazel_command = ["bazel", "test", "--config=ci"] + test_targets
+    command = "\n".join(pre_test_commands + [" ".join(bazel_command)])
+    return subprocess.Popen(_get_docker_run_command() + ["/bin/bash", "-ice", command])
+
+
+def run_command(command: str) -> bytes:
+    """
+    Run command in docker
+    """
+    return subprocess.check_output(
+        _get_docker_run_command() + ["/bin/bash", "-ice", command]
+    )
+
+
+def docker_login() -> None:
+    """
+    Login to docker with AWS credentials
+    """
+    subprocess.run(["pip", "install", "awscli"])
+    p = subprocess.Popen(
+        ["aws", "ecr", "get-login-password", "--region", "us-west-2"],
+        stdout=subprocess.PIPE,
+    )
```

Comment:
just `check_output()` and send the output in on the next one?

otherwise this might be leaking / hanging, sending in empty string other one.